games-strategy/augustus/augustus-3.0.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake readme.gentoo-r1
+
+DESCRIPTION="Feature-enhanced open source re-implementation of Caesar III"
+HOMEPAGE="https://github.com/Keriew/augustus"
+SRC_URI="https://github.com/Keriew/augustus/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+IUSE="uncapped"
+
+DEPEND="
+	media-libs/libpng:0=
+	media-libs/libsdl2[joystick,video,sound]
+	media-libs/sdl2-mixer
+"
+RDEPEND="
+	${DEPEND}
+"
+
+src_prepare() {
+	use uncapped && eapply "${FILESDIR}"/${PN}-3.0.1-uncapped.patch
```

Comment:
Where's the patch? I guess omitted the entire files/ dir :)

Haven't seen it so I can't really tell yet, but what does this do? From the sound of the USE, is this some kind of game cheat? Is it really needed? Users can always use `/etc/portage/patches` for these kind of things -- nevermind if I'm misinterpreting what this does (unfortunately haven't played this game yet, so description doesn't say much to me).

On a side-note, USE-based conditional patches are generally frowned upon (notably given they're not upstream-able, and adding a autoconf/cmake option is preferable). Not that changing is *essential* to get this merged, but well. Depending on the above, may prefer the patch just removed if that's possible.